onnx_tf/handlers/backend/scan_mixin.py

```diff
@@ -0,0 +1,188 @@
+import tensorflow as tf
+from onnx.helper import make_opsetid
+import onnx_tf
+from onnx_tf.common import data_type
+
+
+class ScanMixin(object):
+
+    @classmethod
+    def scan(cls, node, input_dict, strict):
+        current_opset = [make_opsetid(cls.DOMAIN, cls.SINCE_VERSION)]
```

Comment:
This current_opset is the opset version for the whole "body"(graph), it should be using the cls.VERSION. cls.SINCE_VERSION should only be use to determine the version of Scan handler it is going to use.

Reply:
fixed